source/mem_map/mem_map.cpp

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2013-2016, ARM Limited, All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "uvisor-lib/uvisor-lib.h"
+#include "mbed.h"
+#include "main-hw.h"
+#include "mem_map.h"
+#include "plat_mem_map.h"
+
+const MemMap* memory_map_name(uint32_t addr)
+{
+    int i;
+    const MemMap *map;
+
+    /* find system memory region */
+    map = g_mem_map;
+    for(i = 0; i < UVISOR_ARRAY_COUNT(g_mem_map); i++)
+        if((addr >= map->base) && (addr <= map->end))
+            return map;
```

Comment:
same goes for `if` statements - braces ARE mandatory for all block  statements - also for single line cases